docs/pages/database-access/enroll-aws-databases/redshift-serverless.mdx

```diff
@@ -0,0 +1,363 @@
+  enabled: "no"
+proxy_service:
+  enabled: "no"
+```
+
+Replace `example.teleport.sh:443` with the host and port of your Teleport Proxy
+Service. Replace `WORKGROUP_NAME` and `ENDPOINT_NAME` with the names of your
+Redshift Serverless workgroup and endpoint, and replace
+`REDSHIFT_SERVERLESS_URI` with the domain name and port of your Redshift
+Serverless database.
+
+### Bootstrap access to Redshift Serverless
+
+Teleport can bootstrap IAM permissions for the Database Service based on its
+configuration using the `teleport db configure bootstrap` command. You can use
+this command in automatic or manual mode:
+
+- In automatic mode, Teleport will attempt to create appropriate IAM policies
+  and attach them to the specified IAM identity (user or role). This requires the
+  IAM permissions we applied to the `teleport-redshift-serverless-node` role earlier.
+- In manual mode, Teleport will print required IAM policies. You can then create
+  and attach them manually using the AWS management console.
+
+<Admonition type="note" title="AWS credentials">
+AWS Credentials are only required if you’re running the command in "automatic"
+mode. The command uses the default credential provider chain to find AWS
+credentials. The steps in this guide provide AWS credentials to the node by
+applying the `teleport-redshift-serverless-node` role to the instance. See
+[Specifying Credentials](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials)
+for alternative methods of providing AWS credentials for non-EC2 nodes.
+</Admonition>
+
+Run one of the following commands on your Database Service node:
+
+<Tabs>
+<TabItem label="Automatic">
+
+Use this command to bootstrap the permissions automatically when your Teleport
+Database Service runs on a host with AWS credentials. Teleport will automatically
+determine the available role/user, or you can specify one with `--attach-to-role`
+or `attach-to-user`.
+
+```code
+$ teleport db configure bootstrap -c /etc/teleport.yaml
+```
+
+</TabItem>
+<TabItem label="Manual">
+
+Use this command to display required IAM policies which you will then create in your AWS console:
+
+```code
+$ teleport db configure bootstrap -c /etc/teleport.yaml --manual
+```
+
+</TabItem>
+</Tabs>
+
+See the full `bootstrap` command
+[reference](../../database-access/reference/cli.mdx#teleport-db-configure-bootstrap)
+for more information.
+
+### Start the Database service
+
+(!docs/pages/includes/start-teleport.mdx service="the Teleport Database Service"!)
+
+## Step 5/5. Connect
+
+Once the Database Service has started and joined the cluster, log in to see the
+registered databases. Replace `--proxy` with the address of your Teleport Proxy
+Service or cloud tenant:
+
+```code
+$ tsh login --proxy=mytenant.teleport.sh --user=alice
+$ tsh db ls
+Name        Description                    Labels
+----------- ------------------------------ --------
+my-redshift Redshift cluster in us-east-1  ...
```

Comment:
The description here could be a problem for all updated AWS guides. `xxxx in us-east-1` description is added by auto-discovery. With `teleport db configure create`, the description should be empty.